src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -23,10 +26,69 @@
 
     computed: {
       ...mapState('flow', ['flows']),
+
+      canvasHeight() {
+        const defaultHeight = window.screen.availHeight
+        if (!this.flows[0].blocks && this.flows[0].blocks.length) {
+          return defaultHeight
+        }
+
+        const blockAtTheLowestPosition = lodash.maxBy(this.flows[0].blocks, 'platform_metadata.io_viamo.uiData.yPosition')
+
+        if (!blockAtTheLowestPosition) {
+          return defaultHeight
+        }
+
+        const blockElement = document.getElementById(`block/${blockAtTheLowestPosition.uuid}`)
+
+        if (!blockElement) {
+          return defaultHeight
+        }
+
+        const blockHeight = blockElement.clientHeight
+        return lodash.get(blockAtTheLowestPosition, 'platform_metadata.io_viamo.uiData.yPosition') + blockHeight + 100
+      },
+
+      canvasWidth() {
+        const defaultWidth = window.screen.availWidth
+        if (!this.flows[0].blocks && this.flows[0].blocks.length) {
+          return defaultWidth
+        }
+
+        const blockAtTheFurthestRightPosition = lodash.maxBy(this.flows[0].blocks, 'platform_metadata.io_viamo.uiData.xPosition')
+
+        if (!blockAtTheFurthestRightPosition) {
+          return defaultWidth
+        }
+
+        const blockElement = document.getElementById(`block/${blockAtTheFurthestRightPosition.uuid}`)
+
+        if (!blockElement) {
+          return defaultWidth
+        }
+
+        const blockWidth = blockElement.clientWidth
+        return lodash.get(blockAtTheFurthestRightPosition, 'platform_metadata.io_viamo.uiData.xPosition') + blockWidth + 100
+      },
     },
 
     mounted() {
       this.$store.dispatch('builder/loadFlow')
+
+      // Auto scroll, needed to put in mounted()
```

Comment:
@bzabos , I thought about the auto scroll you mentioned during the demo and I found an idea to implement it